### ISSUE TITLE
when i change main.ts  throw error  flash back window

### ISSUE THREAD

**Issue by icyfe** [yhirose/react-typescript-electron-sample-with-create-react-app-and-electron-builder#9]
node ： 12.14.1 
electorn: 9.0.0  
platform: winx64
Hi~ when i change main.ts  throw error  flash back window,

![image](https://user-images.githubusercontent.com/44893721/91691402-2535df80-eb9a-11ea-966e-cf489d1aa120.png)

 can you help for me look forward to your answer!

**Comment by yteruel31** [comment]
Remove `'..',` at hot reloading bloc.

The treatment should be like this (./electron/main.ts) :

```typescript
// Hot Reloading
  if (isDev) {
    // 'node_modules/.bin/electronPath'
    require('electron-reload')(__dirname, {
      electron: path.join(__dirname, '..', 'node_modules', '.bin', 'electron'),
      forceHardReset: true,
      hardResetMethod: 'exit'
    });
  }
```

**Comment by Romex91** [comment]
> Remove `'..',` at hot reloading bloc.
> 
> The treatment should be like this (./electron/main.ts) :
> 
> ```ts
> // Hot Reloading
>   if (isDev) {
>     // 'node_modules/.bin/electronPath'
>     require('electron-reload')(__dirname, {
>       electron: path.join(__dirname, '..', 'node_modules', '.bin', 'electron'),
>       forceHardReset: true,
>       hardResetMethod: 'exit'
>     });
>   }
> ```

Didn't work for me. 

What did work is replacing 'electron' with 'electron.cmd':

``` 
// Hot Reloading

  if (isDev) {
    require("electron-reload")(__dirname, {
      electron: path.join(
        __dirname,
        "..",
        "..",
        "node_modules",
        ".bin",
        "electron" + (process.platform === "win32" ? ".cmd" : "")
      ),
      forceHardReset: true,
      hardResetMethod: "exit",
    });
  }
```

**Comment by yhirose** [comment]
Fixed at #26.
